client/src/components/Filters.js

```diff
@@ -0,0 +1,142 @@
+import React, {useState} from "react";
+import { Dropdown, Form, Radio } from "semantic-ui-react";
+import "./css/filters.css";
+
+function Filters(props) {
```

Comment:
I would suggest making each Filter input group itself a component, and this component is just a collection of those